tf-modules/ingest/discover-granules-task.tf

```diff
@@ -15,7 +15,8 @@ resource "aws_lambda_function" "discover_granules_task" {
     variables = {
       stackName                     = var.prefix
       GranulesTable                 = var.dynamo_tables.granules.name
-      CUMULUS_MESSAGE_ADAPTER_DIR = "/opt/"
+      CUMULUS_MESSAGE_ADAPTER_DIR   = "/opt/"
+      NO_DEPRECATION_WARNINGS       = var.enable_deprecation_warnings
```

Comment:
Hmmm. This looks off, and the names should be consistent. Consider changing `enable_deprecation_warnings` to `no_deprecation_warnings` and set its default value to `false`.

Alternatively, change `NO_DEPRECATION_WARNINGS` to `ENABLE_DEPRECATION_WARNINGS` and set the default value of `enable_deprecation_warnings` to `true`.

Either way will keep default behavior consistent with current behavior.